### ISSUE TITLE
It is not possible to send non-anonymous polls

### ISSUE THREAD

**Issue by igoracmelo** [mymmrac/telego#114]
### 💬 Telego version

v0.25.0

### 👾 Issue description

Using `SendPoll` with `IsAnonymous = false` won't work, because `IsAnonymous` is set to `omitempty`, so `false` will be omited, and Telegram will use the default value for `IsAnonymous`, which is `true`.
So, in other words, you can't send a non-anonymous poll.

I tried fixing it, but all solutions I thought involves potential breaking changes for current users of `SendPoll` (if any), for example:
1. Changing IsAnonymous to a `*bool`
Probably won't break much because the default will still be anonymous poll.
Only if the user explicitly pass `IsAnonymous: true` that he will notice compile errors.

2. Removing `omitempty` and making default polls not anonymous
It won't give compile time errors, but will make polls behave differently by default.

### 🧐 Code example

```Go
_, err = bot.SendPoll(&tg.SendPollParams{
	ChatID: tg.ChatID{
		ID: u.Message.Chat.ID,
	},
	Question: "is this anonymous?",
	Options:  []string{"yes", "no"},
	IsAnonymous: false,
})
```

**Comment by mymmrac** [comment]
Thanks for reporting this, fixed in [v0.25.1](https://github.com/mymmrac/telego/releases/tag/v0.25.1)

**Comment by mymmrac** [comment]
Example:
```go
msg, err := bot.SendPoll(&telego.SendPollParams{
  ChatID:      tu.ID(chatID),
  Question:    "Test",
  Options:     []string{"Option 1", "Option 2"},
  IsAnonymous: telego.ToPtr(false),
})
```
